src/Monitors/PhotoresistorMonitor.cpp

```diff
@@ -10,24 +10,30 @@ void PhotoresistorMonitor::execute()
     float val = analogRead(constants::photoresistor::pin);
     boolean possible_uncovered = false;
 
```

Comment:
This looks good to me.

Originally I was going to say that you could store a pointer to a SensorReading in the sfr. Then you would just have to dereference the pointer and use that SensorReading when adding/getting a value.

I think your implementation is better before this way both buffers are always up to date.

Reply:
Yeah, that was pretty much my reasoning as well.

Unrelated question, do we think it's worth it (as we integrate the SensorReadings more fully into the state machine) retaining any sort of "testing" mode where we only use instantaneous values for sfr and related changes? Just thinking it might get a little tricky to test with like 5 minute buffers, for example.